### DISCUSSION TITLE
Is it possible to get a hint on a exercise

### DISCUSSION THREAD

**Discussion by Pluppen** [browserengineering/book#471] [Q&A] [answered]
Is it possible to get a hint on how to solve exercises? Currently, I'm struggling a little bit with the Shorthand properties in https://browser.engineering/styles.html#exercises

**Comment by pavpanchekha** [comment, marked as answer]
Sure, happy to give a hint. There are a couple of steps you will need to do in this exercise. First, you'll need to change the parser so that `font: bold italic 100% Times` can be parsed. That involves changing `CSSParser.pair`. Then, once you can parse it, you'll want to treat this shorthand property as if it were multiple pairs, like `font-weight: bold; font-style: italic; font-size: 100%; font-family: Times`. That involves changing `CSSParser.body`.
